examples/lighting-app/esp32/main/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ idf_component_register(PRIV_INCLUDE_DIRS
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/occupancy-sensor-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/on-off-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/operational-credentials-server"
+                      "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ota-requestor"
```

Comment:
Why is this change added here?

Reply:
Anyway, that change should be a part of #13532

Reply:
CI does not pass if it's not updated for all platforms - This PR was prepared already before #13532 was published.

Reply:
@dhrishi pinged you on Slack to discuss further if required

Reply:
>CI does not pass if it's not updated for all platforms - This PR was prepared already before #13532 was published.

@tima-q Didn't realize that CI won't pass. Thanks!